frontend/src/action/auth-actions.js

```diff
@@ -1,37 +1,48 @@
 import superagent from 'superagent';
+import * as utils from '../lib/util';
 
-export const tokenSet = token => {
+/********************************************************************************
+*         Synchronous                                                           *
+********************************************************************************/
+
+export const login = token => {
   return {
-    type: 'TOKEN_SET',
```

Comment:
Looks like you renamed your token actions from TOKEN_SET and TOKEN_DELETE to LOGOUT and LOGIN but you did not make those changes in your reducer...oops!